src/internal/blockchain/node/Manager.hpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2010-2022 The Open-Transactions developers
```

Comment:
Another thoughtless name, this is a second Manager.hpp here. What does the class 'manage'? Would it not be more useful to reflect its nature as an abstract interface both in the class name and header?

Reply:
MT-85